placements.sublime-workspace

```diff
@@ -436,10 +436,29 @@
 	},
 	"expanded_folders":
 	[
-		"/Users/Alicia/Desktop/CTD/placements"
+		"/Users/Alicia/Desktop/CTD/placements",
+		"/Users/Alicia/Desktop/CTD/placements/app",
+		"/Users/Alicia/Desktop/CTD/placements/app/controllers",
+		"/Users/Alicia/Desktop/CTD/placements/app/models",
```

Comment:
Alik, I am not sure what this file is, but it looks to be part of your sublime workspace and not something you want to include in your pull request.

Reply:
OK, I moved them. They were workspace files.